System/EnemyArchetypes.ini

```diff
@@ -2160,7 +2160,7 @@ BackupWeapon=(Weapon="SwatEquipment.Glock9mmHG",Chance=20)
 BackupWeapon=(Weapon="SwatEquipment.Glock19HG",Chance=20)
 BackupWeapon=(Weapon="SwatEquipment.BadguyColtM1911HG",Chance=20)
 BackupWeapon=(Weapon="",Chance=20)
```

Comment:
what about this? here is no weapon at all

Reply:
Yes, there is a chance for no secondary weapon.